docs/nginx-ingress-controller.md

```diff
@@ -131,3 +133,9 @@ spec:
 | --- | --- | --- | --- |
 | `enable` | `boolean` | Enable Prometheus metrics. | Yes |
 | `port` | `int` | Sets the port where the Prometheus metrics are exposed. Default is 9113. Format is `1023 - 65535`. | No |
+
+## NginxIngressController.AppProtect
+
+| Field | Type | Description | Required |
+| --- | --- | --- | --- |
+| `enable` | `boolean` | Enable App Protect. | Yes |
```

Comment:
does it make sense to add this and the parent field to the example YAML below ( the one with `The following example shows the usage of all fields (required and optional):` ?
same for `nginxReloadTimeout`